.github/workflows/mac.yml

```diff
@@ -150,7 +150,7 @@ jobs:
           brew install gnu-tar
       - name: Build (${{ inputs.profile }})
         run: |
-          ./mach build --use-crown --locked --${{ inputs.profile }}  ${{ inputs.build-args }}
+          ./mach build --features native-bluetooth --use-crown --locked --${{ inputs.profile }}  ${{ inputs.build-args }}
```

Comment:
Let's not do this for the moment. I'm not sure what the repercussions of shipping this in our nightly build. It's enough to simply fix the build error, I think.